plugins/net_plugin/net_plugin.cpp

```diff
@@ -659,35 +615,35 @@ namespace eosio {
       void set_state(stages s);
       bool sync_required();
       void send_handshakes();
-      bool is_active(connection_ptr conn);
-      void reset_lib_num(connection_ptr conn);
-      void request_next_chunk(connection_ptr conn = connection_ptr() );
-      void start_sync(connection_ptr c, uint32_t target);
-      void reassign_fetch(connection_ptr c, go_away_reason reason);
-      void verify_catchup(connection_ptr c, uint32_t num, block_id_type id);
-      void rejected_block(connection_ptr c, uint32_t blk_num);
-      void recv_block(connection_ptr c, const block_id_type &blk_id, uint32_t blk_num);
-      void recv_handshake(connection_ptr c, const handshake_message& msg);
-      void recv_notice(connection_ptr c, const notice_message& msg);
+      bool is_active(const connection_ptr& conn);
+      void reset_lib_num(const connection_ptr& conn);
+      void request_next_chunk(const connection_ptr& conn = connection_ptr());
+      void start_sync(const connection_ptr& c, uint32_t target);
+      void reassign_fetch(const connection_ptr& c, go_away_reason reason);
+      void verify_catchup(const connection_ptr& c, uint32_t num, const block_id_type& id);
+      void rejected_block(const connection_ptr& c, uint32_t blk_num);
+      void recv_block(const connection_ptr& c, const block_id_type& blk_id, uint32_t blk_num);
+      void recv_handshake(const connection_ptr& c, const handshake_message& msg);
+      void recv_notice(const connection_ptr& c, const notice_message& msg);
    };
 
    class dispatch_manager {
    public:
       uint32_t just_send_it_max = 0;
```

Comment:
Can't `just_send_it_max` now be removed? Along with the config option `max-implicit-request`.

Reply:
Yes. However, since it is a config option and we now error on specified config options that do not exist, I left it. I'm open to removing it. Thoughts?

Reply:
I changed the help text. But I'm certainly open to removing it. What do you think?

Reply:
Let's just remove it.